elastic/datadog_checks/elastic/elastic.py

```diff
@@ -380,6 +385,37 @@ def _process_policy_data(self, data, version, base_tags):
             for metric, desc in iteritems(slm_stats):
                 self._process_metric(policy_data, metric, *desc, tags=tags)
 
+    def _process_cat_allocation_data(self, admin_forwarder, version, base_tags):
+        if version < [7, 2, 0]:
+            self.log.warning(
```

Comment:
```suggestion
            self.log.debug(
```
perhaps more appropriate for debug if erroneously enabled